server/datastore/mysql/vpp.go

```diff
@@ -67,7 +67,9 @@ func (ds *Datastore) GetSummaryHostVPPAppInstalls(ctx context.Context, teamID *u
 ) {
 	var dest fleet.VPPAppStatusSummary
 
-	// TODO(uniq): do we need to handle host_deleted_at and removed similar to GetSummaryHostSoftwareInstalls?
+	// TODO(sarah): do we need to handle host_deleted_at similar to GetSummaryHostSoftwareInstalls?
+	// Currently there is no host_deleted_at in host_vpp_software_installs, so
+	// not handling it as part of the unified queue work.
```

Comment:
I did add the `removed` condition as for software installs (I don't see any reason this should be different here), but I renamed the TODO to add your name Sarah as this is otherwise not related to the uniq work. Let me know if that works for you, or if you prefer that I remove it altogether.